### PR TITLE
Improved regex for urls

### PR DESCRIPTION
**Pull request by wachterjohannes**
Improved regex for URLs.

fixes sulu/sulu#2174

### CODE REVIEW

File: husky_components/input/input-type.js [hunk 1]
```diff
@@ -9,8 +9,8 @@
  */
 
 define([
-    'type/default'
-], function(Default) {
+    'type/default','services/husky/url-validator'
```
Comment (danrot) [inline husky_components/input/input-type.js]: Don't we usually put these into separate lines?

File: husky_components/input/input-type.js [hunk 2]
```diff
@@ -9,8 +9,8 @@
...
+], function(Default, UrlValidator) {
```
Comment (danrot) [inline husky_components/input/input-type.js]: Didn't we agree to start services with lower letters, since they are not constructors?

Comment (wachterjohannes) [inline husky_components/input/input-type.js]: your right (:

**Comment by wachterjohannes** [comment]
@danrot fixed

**Pull request merged**